src/components/Login-class.js

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from "react";
+import styled from "styled-components";
+import {
+  FormControl,
+  FormGroup,
+  Button,
+  FormLabel,
+  Image,
+} from "react-bootstrap";
+import PCTLogo from "../assets/PCT-logo.png";
+
+const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background: #eee;
+`;
+const Title = styled.h2`
+  margin: 10;
+`;
+
```

Comment:
styled components are great!! so glad you discovered this on your own, unfortunately we'll probably be shifting away from styled components for the moment just to keep things simple, can you convert this to standard css?